libs/features/texts/src/npc-text/select-npc-text.component.html

```diff
@@ -19,12 +19,54 @@
           </div>
+          <div class="form-group col-sm-3 col-md-2">
+            <input [formControlName]="'text1_0'" class="form-control form-control-sm" id="search-text1_0" placeholder="text1_0" />
+          </div>
+          <div class="form-group col-sm-3 col-md-2">
+            <input [formControlName]="'text1_1'" class="form-control form-control-sm" id="search-text1_1" placeholder="text1_1" />
+          </div>
+          <div class="form-group col-sm-3 col-md-2">
+            <input [formControlName]="'text2_0'" class="form-control form-control-sm" id="search-text2_0" placeholder="text2_0" />
+          </div>
+          <div class="form-group col-sm-3 col-md-2">
+            <input [formControlName]="'text2_1'" class="form-control form-control-sm" id="search-text2_1" placeholder="text2_1" />
+          </div>
+          <div class="form-group col-sm-3 col-md-2">
+            <input [formControlName]="'text3_0'" class="form-control form-control-sm" id="search-text3_0" placeholder="text3_0" />
+          </div>
+          <div class="form-group col-sm-3 col-md-2">
+            <input [formControlName]="'text3_1'" class="form-control form-control-sm" id="search-text3_1" placeholder="text3_1" />
+          </div>
+          <div class="form-group col-sm-3 col-md-2">
+            <input [formControlName]="'text4_0'" class="form-control form-control-sm" id="search-text4_0" placeholder="text4_0" />
+          </div>
+          <div class="form-group col-sm-3 col-md-2">
+            <input [formControlName]="'text4_1'" class="form-control form-control-sm" id="search-text4_1" placeholder="text4_1" />
+          </div>
+          <div class="form-group col-sm-3 col-md-2">
+            <input [formControlName]="'text5_0'" class="form-control form-control-sm" id="search-text5_0" placeholder="text5_0" />
+          </div>
+          <div class="form-group col-sm-3 col-md-2">
+            <input [formControlName]="'text5_1'" class="form-control form-control-sm" id="search-text5_1" placeholder="text5_1" />
+          </div>
+          <div class="form-group col-sm-3 col-md-2">
+            <input [formControlName]="'text6_0'" class="form-control form-control-sm" id="search-text6_0" placeholder="text6_0" />
+          </div>
+          <div class="form-group col-sm-3 col-md-2">
+            <input [formControlName]="'text6_1'" class="form-control form-control-sm" id="search-text6_1" placeholder="text6_1" />
+          </div>
+          <div class="form-group col-sm-3 col-md-2">
+            <input [formControlName]="'text7_0'" class="form-control form-control-sm" id="search-text7_0" placeholder="text7_0" />
+          </div>
+          <div class="form-group col-sm-3 col-md-2">
+            <input [formControlName]="'text7_1'" class="form-control form-control-sm" id="search-text7_1" placeholder="text7_1" />
+          </div>
```

Comment:
I would opt for a `for-of ['text0_1', 'text1_0' ....]`

Reply:
you could use the `NPC_TEXT_SEARCH_FIELDS` array removing the first element to do this for-of